test/lib/helpers/FakeWeb3Provider.ts

```diff
@@ -0,0 +1,92 @@
+import assert from 'assert';
+
+interface RPCResponse {
+  jsonrpc: string;
+  id: number;
+  result?: object;
+}
+
+interface RPCError {
+  jsonrpc: string;
+  id: number;
+  error: object;
+}
+
+interface Dictionary<T> {
+  [key: string]: Array<T>;
+}
+
+export default class ExternalProvider {
+  private requestId: number = 0;
```

Comment:
Can we change that linter behaviour? IMO we should not throw if we provide a type that can be automatically inferred. What do you guys think?

Reply:
I would also agree on allowing (or even enforcing) to write the type explicitly.

Reply:
I disabled this behavior in the latest commit at least for class properties and function parameters. I kind of do agree that
 
```
function foo() {
  const a: number = 1;
}
```

looks quite funny so I set a warning for this.